static/css/redox.css

```diff
@@ -2,18 +2,23 @@
   box-sizing: border-box;
 }
 
-head, body {
-  font-family: 'Lato', 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+a {
+  font: Helvetica, arial, nimbussansl, liberationsans,freesans, clean, sans-serif
```

Comment:
IMHO Lato looks better and for sure Lato has better license and is available on all platforms.

Also it is 3rd most popular font according to Google Web Fonts.

BTW I lato (summer in polish and I'm Pole).